test/conf_include_test.c

```diff
@@ -37,28 +37,32 @@
 #endif
 
 /* changes path to that of the filename */
-static int change_path(const char *file)
+static char * change_path(const char *file)
```

Comment:
Our coding style doesn't have a space after the char *.

Reply:
Thanks for catching, fixed.